cmd/mattermost/commands/channel.go

```diff
@@ -80,12 +80,12 @@ Archived channels are appended with ' (archived)'.`,
 }
 
 var MoveChannelsCmd = &cobra.Command{
-	Use:   "move [team] [channels]",
+	Use:   "move [team] [channels] [flags]",
```

Comment:
Is `[flags]` obvious to the admin? Perhaps `move [team] [channels] --username [user]` is move obvious? 0/5

Reply:
agree. done